l1-contracts/package.json

```diff
@@ -57,7 +58,8 @@
   "scripts": {
     "build": "hardhat compile && CONTRACTS_BASE_NETWORK_ZKSYNC=true hardhat compile ",
     "build-l1": "hardhat compile",
-    "build:foundry": "forge build && forge build --zksync --skip '*/l1-contracts/test/*'",
+    "build:foundry": "forge build --skip contracts/bridgehub/L1Bridgehub.sol && forge build --optimizer-runs 28000 contracts/bridgehub/L1Bridgehub.sol && forge build --zksync --skip '*/l1-contracts/test/*'",
```

Comment:
flagging this, we should remove if possible. It messes up the whole zkstack deployment speed